### PR TITLE
Fix 164: patch the way to convert NullableArrays

### PR DESCRIPTION
**Pull request by randy3k**
Patch the way to convert NullableArrays and improve the corresponding tests.

Fix 164.

### CODE REVIEW

File: test/dataframe.jl [hunk 1]
```diff
@@ -13,33 +13,51 @@ v110 = rcopy(NullableArray,reval("c(1L, NA)"))
...
+v = NullableArray([1,2], [false,true])
+@test isequal(rcopy(NullableArray,RObject(v)), v)
+@test isequal(rcopy(NullableArray,"c(NA,1)"), NullableArray([0,1.], [true,false]))
+@test isequal(rcopy(NullableArray,"c(1,NA)"), NullableArray([1.,0], [false,true]))
+# real
 v = NullableArray([1.,2.], [true,false])
 @test isequal(rcopy(NullableArray,RObject(v)), v)
+v = NullableArray([1.,2.], [false,true])
+@test isequal(rcopy(NullableArray,RObject(v)), v)
+@test isequal(rcopy(NullableArray,"c(NA,1L)"), NullableArray([0,one(Int32)], [true,false]))
```
Comment (nalimilan) [inline test/dataframe.jl]: I think you confused the integer and numeric cases (see previous block).

Comment (randy3k) [inline test/dataframe.jl]: 👍 good catch, I have fixed it.

**Comment by randy3k** [comment]
It seems CategoricalArrays is corresponding to the failure of the nightly build.

**Comment by nalimilan** [comment]
Yes, that's a known issue (bug in Julia AFAICT).

**Pull request merged**